### ISSUE TITLE
Get weekly expiry date also

### ISSUE THREAD

**Issue by ChillarAnand** [swapniljariwala/nsepy#38]
BANKNIFTY has weekly expiry. It would be good to have weekly expirty also.

**Comment by mbmarx** [comment]
Weekly expiry should be supported to make most use of this NSEPy. Can this be considered

**Comment by mbmarx** [comment]
@swapniljariwala any solution to weekly expiry data download

**Comment by ChillarAnand** [comment]
@mbmarx I don't think this repo is actively maintained anymore. Last commit was ~3 years ago.
